integrations/extensions/starter-kits/ibm-rpa/README.md

```diff
@@ -0,0 +1,113 @@
+3. For security purposes go to the namespace section of cloud function and get the GUID.
+4. Know the API URL for your RPA Tenant (us1/BR2,UK1, etc).
+5. Know your RPA username and passowrd.
+
+Next create a cloud function used to insert an item in the RPA queue:
+
+1. Open the [cfInsert.js](./cfInsert.js) template file included with this starter kit in the text editor of your choice.
+2. Write your personal values from above into the top of the cfInsert.js template script.
+3. The following parameters should be updated based on your RPA setup above:
+![cfInsert-params.png](./assets/cfInsert-params.png)
+4. Build dynamic logic to insert the RPA expected variables (must match RPA script variables defined above). This involves modifying the script file variable names to match your RPA script (e.g., rpaInputParam1).
+5. Create your action and name it "Start RPA Asynchronously"
+![start-rpa-asyncrhonously](./assets/start-rpa-asynchronously.png)
+6. Copy the edited javascript from the cfInsert.js file into the code section of the action.
+7. Go to Endpoints and enable endpoint as a Web Action (.json will be used by the Extension to define the response format).
+
+Next create the cloud function for reading status of an item in the queue:
+1. Open the [cfGetStatus.js](./cfGetStatus.js) template file included with this starter kit in the text editor of your choice.
+2. Write your personal values from above into the top of the cfGetStatus.js template script.
+3. The following parameters should be updated based on your RPA setup above:
+![cfGetStatus-params.png](./assets/cfGetStatus-params.png)
+5. Create your action and name it "Get RPA Asynchronous Status"
+![get-rpa-asynchronous-status](./assets/get-rpa-asynchronous-status.png)
+6. Copy the edited javascript from the cfInsert.js file into the code section of the action.
+7. Go to Endpoints and enable endpoint as a Web Action (.json will be used by the Extension to define the response format).
+
+Next update the OpenAPI spec template included with this starter kit:
+1. Open the [ibm-rpa.openapi.json](./ibm-rpa.openapi.json) template file included with this starter kit in the text editor of your choice.
+2. Copy and paste over necessary lines in the template with the following based on what your RPA functions use for input and output parameters (e.g., rpaInputParam1):
+![open-api-parameters](./assets/open-api-parameters.png)
+3. Save changes
+
+### Setup Watson Assistant
+
+1. Within the new Watson Assistant environment, start by creating a new assistant. You can do this by importing the ibm-rpa.actions.json file. Note that the parameters and endpoints related to the RPA extension defined within this JSON file will have to be setup again after you import the Extension. 
```

Comment:
I suggest not to use the term `environment`

Reply:
Changed environment to instance.